common/presenters/events-to-timeline/add-triggered-events.test.js

```diff
@@ -0,0 +1,43 @@
+const i18n = require('../../../config/i18n')
+
+const addTriggeredEvents = require('./add-triggered-events')
+
+const mockEvents = [
+  { event_type: 'foo', value: 1 },
+  { event_type: 'bar', value: 2 },
+]
+
+describe('Timeline events', function () {
+  describe('#addTriggeredEvents', function () {
+    let events
+    beforeEach(function () {
+      events = {}
+      sinon.stub(i18n, 't').returnsArg(0)
+      sinon.stub(i18n, 'exists').returns(false)
+    })
+
+    context('When there are no triggered events', function () {
+      beforeEach(function () {
+        events = addTriggeredEvents(mockEvents)
+      })
+      it('should leave the events untouched', function () {
+        expect(events).to.deep.equal(mockEvents)
+      })
+    })
+
+    context('When there are triggered events', function () {
```

Comment:
Could we get a test for a mix of triggered and untriggered events?  That way we can be sure that the filter code is doing the correct job, and not just returning if it finds an untriggered event.

Reply:
Done